erpnext/manufacturing/doctype/production_plan/production_plan.py

```diff
@@ -212,13 +212,16 @@ def delete_draft_work_order(self):
 			filters = {'docstatus': 0, 'production_plan': ("=", self.name)}):
 			frappe.delete_doc('Work Order', d.name)
 
-	def set_status(self):
+	def set_status(self, update=None):
```

Comment:
```suggestion
	def set_status(self, update=None):
```
Update parameter is not used.